.github/workflows/comment-pr-with-useful-links.yml

```diff
@@ -0,0 +1,11 @@
+name: Comments PR with useful links
```

Comment:
**praise:** we need to decide if we keep this wf
if so, it should probably renamed, we should run it only on some conditions (when the underlying reusable workflow or internal actions change). It should probably also be run on pull_request_target for consistency with the usage of the reusable wf in the content repositories
The wf will never create comments but running it may help detecting issue early

Reply:
I think we can keep it to check faster if we break something in reusable workflow, composite action, or final workflow.
However, I will restrict the run and the target of this workflow.

Reply:
Rename the file and change the name (see reusable workflow)